### DISCUSSION TITLE
how do you disable replay option?

### DISCUSSION THREAD

**Discussion by EnvyBun** [KadeArchive/Kade-Engine#106] [Q&A] [answered]
im trying to disable the replay option because my game crashes every time i finish a song (modded in or not)could someone show how to disable the replay option? i mean there is a replay.hx file but idk if delete it it will break the game

**Comment by puyoxyz** [comment, marked as answer]
you can't disable replays, the reason your game is crashing is that you don't have a `replays` folder in `assets`, or you don't have permission to access it. if the `replays` folder exists, make sure the game is not in program files or in another user's directory (i.e. another user's desktop)
